travis/travis_install_nightly

```diff
@@ -1,6 +1,6 @@
 #!/bin/bash
 
-pip install -q QUnitSuite flake8 coveralls pylint Click > /dev/null 2>&1
+pip install -q QUnitSuite flake8 coveralls Click oca-pylint-plugin > /dev/null 2>&1
```

Comment:
@bwrsandman 
I suggest create webhook project and was approved 3 mothns ago.
http://awesomescreenshot.com/03a57yz56a
Today we don't have created the project.
Create project is slow.

If we wait a new project the process is:
1. Create project
  1.1 Discuss 3 weeks
  1.2 Approved and creation of project 2 weeks.
2. First PR
  2.1 First PR and first feedback 1 week.
  2.1 Feedback and fix iteration (big code) 8 weeks.

We can run this slow and large process in parallell.
I sent a email with your good proposal of create a new project.
But help me to push this PR before of this process, I promisse a 100% of coverage and green travis in my plugin.